vms/platformvm/block/builder/helpers_test.go

```diff
@@ -205,12 +181,12 @@ func addSubnet(t *testing.T, env *environment) {
 	testSubnet1, err = env.txBuilder.NewCreateSubnetTx(
 		2, // threshold; 2 sigs from keys[0], keys[1], keys[2] needed to add validator to this subnet
 		[]ids.ShortID{ // control keys
-			preFundedKeys[0].PublicKey().Address(),
-			preFundedKeys[1].PublicKey().Address(),
-			preFundedKeys[2].PublicKey().Address(),
+			ts.Keys[0].PublicKey().Address(),
```

Comment:
if we keep `testSubnet1ControlKeys` these here should be `testSubnet1ControlKeys`.
Otherwise we should drop `testSubnet1ControlKeys`